From pavel@ucw.cz  Mon Jul 27 08:35:44 2009
From: Pavel Machek <pavel@ucw.cz>
Date: Tue, 21 Jul 2009 12:38:46 +0200
Subject: Staging: htc: Dream: limit Kconfig for only MSM platforms
To: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Cc: Greg KH <greg@kroah.com>
Message-ID: <20090721103846.GH4133@elf.ucw.cz>
Content-Disposition: inline

Limit SMD communication glue to MSM platform. It is closely tied to
MSM architecture.

Signed-off-by: Pavel Machek <pavel@ucw.cz>
Cc: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/dream/smd/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/dream/smd/Kconfig
+++ b/drivers/staging/dream/smd/Kconfig
@@ -1,4 +1,5 @@
 config MSM_SMD
+	depends on ARCH_MSM
 	default y
 	bool "MSM Shared Memory Driver (SMD)"
 	help
